apps/quick-learn-backend/src/entities/course.entity.ts

```diff
@@ -20,7 +20,7 @@ export class CourseEntity extends BaseEntity {
   @Column({ type: 'varchar', length: 5000 })
   description: string;
 
-  @Column({ type: 'int', nullable: false })
+  @Column({ type: 'int', nullable: true })
```

Comment:
This shouldn't be nullable. Course category be deleted only when the user unassigned or changes the of each course it has.